cmd/utils/flags.go

```diff
@@ -131,6 +131,10 @@ var (
 		Usage: "Explicitly set network id (integer)(For testnets: use --ropsten, --rinkeby, --goerli instead)",
 		Value: eth.DefaultConfig.NetworkId,
 	}
+	MainnetFlag = cli.BoolFlag{
+		Name:  "mainnet",
+		Usage: "Ethereum network: main network of the ETH asset",
```

Comment:
Maybe just
```suggestion
		Usage: "Ethereum mainnet",
```
At least drop that "of the ETH asset" :)